scripts/setup.sh

```diff
@@ -1,18 +1,10 @@
-#!/usr/bin/env bash
+#!/bin/sh
 set -e
 
-if ! command -v pdm &>/dev/null; then
-    if ! command -v pipx &>/dev/null; then
-        python3 -m pip install --user pipx
-    fi
-    pipx install pdm
-fi
-if ! pdm self list 2>/dev/null | grep -q pdm-multirun; then
-    pdm install --plugins
-fi
+command -v pipx &>/dev/null || python3 -m pip install --user pipx
+command -v pdm &>/dev/null || pipx install pdm
```

Comment:
The thing is, PDM is not obviously installed through pipx. So while this is more concise, the semantics changed: even if PDM is installed, the script will try to install pipx.